### ISSUE TITLE
Request: Patient.dropItem(item), Tumor.dropItem(item)

### ISSUE THREAD

**Issue by howew** [imsweb/naaccr-xml#146]
I would like to request that functions to drop items from a patient and tumor be added.

This would be useful in the scenario where we are reading patients from an "Abstract" or "Confidential" NAACCR XML file and writing those patients to an "Incidence" NAACCR XML file.  Currently if you attempt to do this it will throw an error because the writer only knows about the Incidence fields.  By adding these functions we can drop the items the writer doesn't know about from the patient and tumor before we attempt to write them.

**Comment by depryf** [comment]
That shouldn't be difficult to add. I might call the method "removeItem(String naaccrId)" since there is already a "addItem(Item item)" one. Just to be consistent.

**Comment by depryf** [comment]
These changes have been implemented. They will be available with the next release of the library.

Every data level (NaaccrData, Patient and Tumor) has a new method "removeItem" that either takes a NAACCR ID, or an Item object.

In addition to that, I also added a "removePatient(idx)" on the NaaccrData class and a "removeTumor(idx)" on the Patient class.  They serve the same purpose as the "removeItem" method: they allow the manipulation of the data model before writing it...
